src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -1154,7 +1154,61 @@ public final class JavadocTokenTypes {
      */
     public static final int P_HTML_TAG_NAME = JavadocParser.P_HTML_TAG_NAME;
 
-    /** List item tag name. */
+    /**
+     * List item tag name.
+     *
+     * <p><b>Example:</b></p>
+     * <pre>{@code
+     *  <ol>
+     *    <li>banana</li>
+     *    <li>apple</li>
```

Comment:
The list contains two elements, while the Javadoc tree doesn't have an `apple` element. Could you make them aligned with each other

btw I reckon one `<li>` element is enough to demonstrate the usage